phabfive/cli.py

```diff
@@ -259,6 +283,56 @@ def run(cli_args, sub_args):
             if sub_args["whoami"]:
                 u.print_whoami()
 
+        if cli_args["<command>"] == "repl":
+            r = repl.Repl()
+            r.run()
+
+        if cli_args["<command>"] == "maniphest":
+            m = maniphest.Maniphest()
+            
+            if sub_args["add"] and sub_args["comment"]:
+                result = m.add_comment(
+                    sub_args['<ticket_id>'],
+                    sub_args['<comment>'],
+                )
+                if result[0]:
+                    # Query the ticket to fetch the URI for it
+                    _, ticket = m.info(int(sub_args["<ticket_id>"][1:]))
+
+                    print("Comment successfully added")
+                    print("Ticket URI: {0}".format(ticket["uri"]))
+
+            if sub_args["info"]:
+                _, result = m.info(int(sub_args["<ticket_id>"][1:]))
+
+                from datetime import datetime
+
+                print("Ticket ID:          {0}".format(result["id"]))
+                print("phid:               {0}".format(result["phid"]))
+                print("authorPHID:         {0}".format(result["authorPHID"]))
+                print("ownerPHID:          {0}".format(result["ownerPHID"]))
+                print("ccPHIDs:            {0}".format(result["ccPHIDs"]))
+                print("status:             {0}".format(result["status"]))
+                print("statusName:         {0}".format(result["statusName"]))
+                print("isClosed:           {0}".format(result["isClosed"]))
+                print("priority:           {0}".format(result["priority"]))
+                print("priorityColor:      {0}".format(result["priorityColor"]))
+                print("title:              {0}".format(result["title"]))
+                print("description:        {0}".format(result["description"]))
+                print("projectPHIDs:       {0}".format(result["projectPHIDs"]))
+                print("uri:                {0}".format(result["uri"]))
+                print("auxiliary:          {0}".format(result["auxiliary"]))
+                print("objectName:         {0}".format(result["objectName"]))
+                print("dateCreated:        {0} ({1})".format(
+                    result["dateCreated"],
+                    datetime.fromtimestamp(int(result["dateCreated"])),
+                ))
+                print("dateModified:       {0} ({1})".format(
+                    result["dateModified"],
+                    datetime.fromtimestamp(int(result["dateModified"])),
+                ))
+                print("dependsOnTaskPHIDs: {0}".format(result["dependsOnTaskPHIDs"]))
+
```

Comment:
Too much info. And too "debuggy" looking. The default output should be more human-friendly, with perhaps only `uri`, `statusName`, and `priority`. On my wishlist is also to list parents and sub-tasks. And the rest of this information should perhaps be hidden behind a `--verbose` flag? IMHO.

Reply:
This was resolved in the latest commit, the items that was decided to remain is now few, the rest is commented out for future implementation of --verbose flag when bigger rewrite will be done to better support the data manipulation through phabfive.